app/controllers/home_controller.rb

```diff
@@ -1,4 +1,41 @@
 class HomeController < ApplicationController
+  
+  
   def index
-  end
-end
+    pasta = Pasta.all
```

Comment:
Though it won't break the code, suggest having a different variable name that stores all the pasta entries and a different variable name that is storing one pasta entry.